docs/JestObjectAPI.md

```diff
@@ -240,6 +240,23 @@ const moduleName = require('../moduleName');
 moduleName(); // Will return '42';
 ```
 
+When using the `factory` parameter for an ES6 module with a default export, the `__esModule: true` property needs to be specified:
```

Comment:
Can you add something about how both Babel's and TypeScript's module transform looks for this property? Without that context, it might look really weird, I think 🙂

Reply:
Actually had to research that :). Does something like that work?

Reply:
Yeah, I think so. Thanks!

Reply:
Pleasure!